pkg/controller/runtime.go

```diff
@@ -113,7 +113,6 @@ type Output struct {
 type Reader interface {
 	Get(context.Context, resource.Pointer, ...state.GetOption) (resource.Resource, error)
 	List(context.Context, resource.Kind, ...state.ListOption) (resource.List, error)
-	WatchFor(context.Context, resource.Pointer, ...state.WatchForConditionFunc) (resource.Resource, error)
```

Comment:
intended?

Reply:
yes... I believe we're not using it anywhere (need to verify though)

Reply:
i thought it was used in talos, not sure

Reply:
it is used indeed, but not in controllers, so gc'ed unused thing, and it's confusing.

Reply:
confirmed that it's not use neither in Talos nor in Omni